pyartcd/pyartcd/pipelines/check_bugs.py

```diff
@@ -184,10 +184,11 @@ async def _slack_report(self):
             for i in report[k]:
                 message += f'\n{i}'
 
-        self.logger.info('Sending notification to Slack')
-        self.logger.debug(message)
-        await self.slack_client.say(message)
 
+            self.logger.info('Sending notification to Slack')
+            self.logger.debug(message)
+            await self.slack_client.say(message)
+            message = ''
```

Comment:
Curious as to why manually setting it to `null` at the end of a function?

Reply:
For sending slack messages individually for each version and not as a collection over all versions